CMakeLists.txt

```diff
@@ -3330,8 +3330,9 @@ endif()
 set_source_files_properties(src/dynapi/SDL_dynapi.c PROPERTIES SKIP_PRECOMPILE_HEADERS 1)
 
 set(SDL_FRAMEWORK_RESOURCES
-  Xcode/SDL/pkg-support/resources/ReadMe.txt
+  Xcode/SDL/pkg-support/resources/INSTALL.md
```

Comment:
This one is not correct.
CMake generates a single-target single-architecture framework, not a xcframework.
```suggestion
```